app/packages/core/src/components/Actions/Options.tsx

```diff
@@ -191,7 +191,6 @@ const MediaFields = ({ modal }) => {
   const [selectedMediaField, setSelectedMediaField] = useRecoilState(
     fos.selectedMediaField(modal)
   );
-
```

Comment:
Can this be omitted?

Reply:
We need it, because we need the `setSelectedMediaField` method to update the `selectedMediaField` state on the modal.

Reply:
I was only referring to line removal. But it looks like good cleanup. Disregard!